package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethereum-ens",
-  "version": "0.7.1",
+  "version": "0.8.0",
```

Comment:
This should probably be 0.7.2, since it's backwards-compatible now.

Reply:
Hmm, semver says 0.x is backwards compatible added features, though.

Reply:
Only if the major version is 1 and above, AIUI - if it's 0, minor releases are breaking.

Reply:
Makes sense. I changed it.